locales/en/plugin__networking-console-plugin.json

```diff
@@ -368,6 +384,7 @@
   "Terminating": "Terminating",
   "Termination type": "Termination type",
   "The format should match standard CIDR notation (for example, \"10.128.0.0/16\").": "The format should match standard CIDR notation (for example, \"10.128.0.0/16\").",
+  "The network connect to a physical netwrok through an Open vSwitch bridge.": "The network connect to a physical netwrok through an Open vSwitch bridge.",
```

Comment:
Correction: 'The network connects to a physical network through an Open vSwitch bridge."